.travis.yml

```diff
@@ -37,7 +37,7 @@ env:
   - TEST_TARGET="ci-fiat-crypto"
   - TEST_TARGET="ci-fiat-parsers"
   - TEST_TARGET="ci-flocq"
-  - TEST_TARGET="ci-hott"
+  - TEST_TARGET="ci-hott" BUILD_TARGET="tools coqbinaries pluginsopt states"
```

Comment:
I don't favor this kind of customization as for a) IMHO it should be contained in the HoTT script overlay; b) we will share the same Coq build anyways soon thanks to Travis build stages.